tests/acceptance/features/coreApiWebdavMove1/moveFolderToBlacklistedName.feature

```diff
@@ -30,7 +30,6 @@ Feature: users cannot move (rename) a folder to a blacklisted name
   Scenario Outline: Rename a folder to a banned name
     Given using <dav_version> DAV path
     And user "Alice" has created folder "/testshare"
-    And the administrator has updated system config key "blacklisted_files" with value '["blacklisted-file.txt",".htaccess"]' and type "json"
```

Comment:
The whole test scenario is useless without the ability do somehow define the `blacklisted_files`
We could put a "ToDo' comment in the scenario to say that a step is needed that defines the blacklisted files, and that step cannot be written yet.

Reply:
@phil-davis Should we keep `ToDo` comment or remove the scenarios and mention the PR to issue https://github.com/owncloud/ocis/issues/1323 ?

Reply:
I think that we should remove all these scenarios. If the features are implemented somehow in oCIS in the future, then we can write new test scenarios that test the exact behavior of the features in oCIS.